meaning/meaning-gui.py

```diff
@@ -0,0 +1,136 @@
+import flet as ft
+import requests
+import bs4
+from bs4 import BeautifulSoup
+
+
+def get_meaning(word, site, datasrc):
```

Comment:
so you made a copy instead of using the pybites-tools dependency because you had to adjust things right?

Reply:
Yes, I suppose I could have left his "meaning.py" as a separate file and called it... Maybe should have done it that way.

Reply:
Yeah sometimes it's a tough balance. Bringing in pybites-tools as a dependency, as I did on clinic, also brings in a whole other bunch of scripts. Plus my flet script with an external file would not build well (still have to root cause that btw). So it is also nice that it's now all in one file.